install/generator/07-syndesis-upgrade.yml.mustache

```diff
@@ -1,7 +1,7 @@
 - apiVersion: v1
   kind: PersistentVolumeClaim
   metadata:
-    name: syndesis-upgrade
+    name: {{ProductName}}-upgrade
```

Comment:
This is a mistake on my part. The intention is to change only the pod names. I will fix this (and other occurrences if any).